packages/notification-managment-spa/src/app/form/form.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { AppService } from '../app.service';
+import { UserProfile } from '../helper';
+
+@Component({
+  selector: 'op-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.scss']
+})
+export class FormComponent implements OnInit {
```

Comment:
Why is it called `FormComponent`? That's not how the components should be named.
Also, the Notifications dashboard has 2 forms, 1 for configuration, and 1 for manual notifications. So how would someone differentiate between the two?

Reply:
This was named differently before

Reply:
Need help on this one

Reply:
@gisumit

Reply:
yes, there are 2 types/types of form in the notification 

1.  Manage Notification 
1.1 New 
1.2 Edit
2 form for manual notification  

I think the name of this component needs to be Manage Notification with **two-state [ new & edit ]**

Reply:
Changed to manage-notification